docs/conf.py

```diff
@@ -31,3 +31,8 @@
 
 # Be strict about any broken references:
 nitpicky = True
+
+extensions += ['sphinx.ext.intersphinx']
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/3', None),
+}
```

Comment:
Given https://github.com/python/importlib_resources/issues/235, I guess this change probably needs to go in the skeleton too.

Reply:
That sounds reasonable.

Reply:
Pushed as jaraco/skeleton@8ea55f2fb26bd77997f0e9435bab2d41376a76d4.